site/content/en/docs/tasks/run_jobs.md

```diff
@@ -19,6 +19,13 @@ Make sure the following conditions are met:
 - [Kueue is installed](/docs/installation).
 - The cluster has [quotas configured](/docs/tasks/administer_cluster_quotas).
 
+<figure>
+  <img src="/images/queueing-components.svg" alt="Kueue Components" style="width:100%">
+  <figcaption>Kueue Components</figcaption>
+</figure>
+
+Kubernetes batch workload is abstracted by a Kueue workload which is submitted to a LocalQueue for scheduling. LocalQueue is a namespace-scoped resource that is created by the user. The LocalQueue is then mapped to a ClusterQueue by the Kueue controller.
```

Comment:
Maybe you can just say: The following picture shows all the concepts you will interact with in this tutorial.